src/HDInsight/HDInsight/ManagementCommands/GrantAzureHDInsightHttpServicesAccessCommand.cs

```diff
@@ -16,10 +16,13 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.HDInsight.Models;
 using Microsoft.WindowsAzure.Commands.Common;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+using System;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
+    [CmdletDeprecation(ReplacementCmdletName = "Set-AzHDInsightGatewayCredential")]
```

Comment:
What do we mean in this case?  Generally, cmdlet deprecation means that the cmdlet will stop workign in the future, and we will remove it before that.  If this cmdlet is currently not working, we should remove it and issue a breaking change notice.

Reply:
> What do we mean in this case? Generally, cmdlet deprecation means that the cmdlet will stop workign in the future, and we will remove it before that. If this cmdlet is currently not working, we should remove it and issue a breaking change notice.

After disscussion, we have removed the two cmdlets rather than deprecated.